ui/src/components/import/panel/ValidationPanel.js

```diff
@@ -134,14 +130,49 @@ const ValidationPanel = (props) => {
               <TableCell>{info.incompleteSurveyCount}</TableCell>
               <TableCell>incomplete surveys found</TableCell>
             </TableRow>
+            <TableRow>
+              <TableCell>{info.siteCount}</TableCell>
+              <TableCell>distinct sites found</TableCell>
+            </TableRow>
+            <TableRow>
+              <TableCell></TableCell>
+              <TableCell>
+                <small>
+                  {info.newSiteCount} new site(s) found
+                  <br />
+                  <i>
+                    {info.foundSites &&
+                      Object.keys(info.foundSites).map((key, value) => {
```

Comment:
I think this was meant to be:
```javascript
                    {info.foundSites &&
                      Object.keys(info.foundSites).map((key) => {
                        return (
                          <div key={key}>
                            {key} {info.foundSites[key] ? '' : '(new)'}
                          </div>
                        );
                      })}
```

Currently its just using the index to determine if its "new"

Reply:
Yes, you're right. Also moved the site code info into a tooltip.